src/framework/tooltip/_tooltip.scss

```diff
@@ -0,0 +1,50 @@
+
+$fieldMessage_arrowSide: 6px;
+$fieldMessage_bgColor: #fafdfe;
+$fieldMessage_borderColor: #dbe2e3;
+
+.tooltip {
+  padding: 15px 13px;
+}
+
+.tooltipParent {
+  display: inline-block;
+  position: relative;
+}
+
+.tooltipPosition {
+  display: none;
+  opacity: 0;
+  position: absolute;
+
+  &:after {
+    background-color: $fieldMessage_bgColor;
+    border: solid $fieldMessage_borderColor;
+    border-radius: 3px;
+    border-width: 1px 0 0 1px;
+    content: '';
+    display: block;
+    height: $fieldMessage_arrowSide;
+    position: absolute;
+    margin-left: -$fieldMessage_arrowSide;
+    margin-top: -4px;
+    transform: rotate(45deg);
+    top: 0;
+    right: 20px;
+    width: $fieldMessage_arrowSide;
+  }
+
+  &.tooltipTop:after {
+    border-width: 0 1px 1px 0;
+    top: initial;
+  }
+
+  &.tooltipLeft:after {
+    left: 20px;
+  }
+
+  .box {
+    box-shadow: 0 2px 4px 0 rgba(189, 189, 189, 0.5);
```

Comment:
The background color of the arrow and of the box is different. The box is still white.